src/lib/builtins/array.rs

```diff
@@ -573,6 +607,7 @@ pub fn create_constructor(global: &Value) -> Value {
     array_prototype.set_field_slice("includes", includes_func);
     array_prototype.set_field_slice("indexOf", index_of_func);
     array_prototype.set_field_slice("lastIndexOf", last_index_of_func);
+    array_prototype.set_field_slice("fill", to_value(fill as NativeFunctionData));
```

Comment:
You need to set the `length` property of a function.

Reply:
@IovoslavIovchev can you please be more specific? I'm afraid I don't understand this comment

Reply:
`Function`s have their `length` property set to the number of required parameters they have (e.g. those with no default value).

You can take a look at how it's currently done [here](https://github.com/bojan88/boa/blob/1728795ff784ffabe3ad88369a9bd2fe7a5b4f3a/src/lib/builtins/array.rs#L596).

Reply:
Thanks @IovoslavIovchev. I didn't know that `length` property existed on this function/object.